readme.md

```diff
@@ -51,6 +51,17 @@ git clone --recursive https://github.com/Azure/azure-iot-sdk-csharp.git
+:heavy_check_mark: feature available  :heavy_multiplication_x: feature planned but not supported  :heavy_minus_sign: no support planned
+
+This repository contains [provisioning device client SDK](./provisioning/device) for the [Device Provisioning Service](https://docs.microsoft.com/en-us/azure/iot-dps/).  Provisioning service SDK for C# is work in progress.  Please see provisoning service SDK in [Node.js](https://github.com/Azure/azure-iot-sdk-node) and [Java](https://github.com/Azure/azure-iot-sdk-java) in the meantime.
+
+| Features                    | mqtt               | mqtt-ws            | amqp               | amqp-ws            | https              | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|-----------------------------|--------------------|--------------------|--------------------|--------------------|--------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| TPM Individual Enrollment   | :heavy_check_mark: | :heavy_check_mark: | :heavy_multiplication_x: | :heavy_check_mark: | :heavy_check_mark: | This SDK supports connecting your device to the Device Provisioning Service via [individual enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#enrollment) using [Trusted Platform Module](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-security#trusted-platform-module-tpm).  Please review the [samples](./provisioning/device/samples/) folder on how to create a device client.    Websocket connection over AMQP is in progress.                                                                                                                                                                                                        |
```

Comment:
>:heavy_check_mark: [](start = 101, length = 18)

This is also not working.

Reply:
@yzhong94 @tameraw TPM over AMQP works (just tested) - sorry for giving you the wrong info earlier today.
/cc @jasmineymlo